website/content/docs/drivers/raw_exec.mdx

```diff
@@ -159,6 +162,15 @@ config {
 }
 ```
 
+- `denied_envvars` - (Optional) Passes a list of environment variables to be
+  scrubbed from all task environments. Supports simple globbing, "*" wildcard
+  accepted as prefix and/or suffix.
```

Comment:
```suggestion
- `denied_envvars` - (Optional) Passes a list of environment variables that
  the driver should scrub from all task environments. Supports globbing "*",
  with wildcard accepted as prefix and/or suffix.
```

style: use active voice; also we don't use words like "simple" so I removed it rather than substitute "basic"

I'm not sure if the driver scrubs the variables or Nomad scrubs the variables